tests/rigsmtr.c

```diff
@@ -394,7 +394,7 @@ int set_conf_rig(RIG *rig, char *conf_parms)
 
         if (!q)
         {
-            return RIG_EINVAL;
```

Comment:
This change is ok because the code is parsing a line was supposed to contain an equal sign but it is not.